### PR TITLE
Compute v2: Implement Hypervisor request options 

### PR DESCRIPTION
**Pull request by jtopjian**
For #2132 
Supersedes #2131

### CODE REVIEW

**Comment by jtopjian** [comment]
recheck

**Comment by jtopjian** [comment]
@kayrus Do you mind reviewing this when you have time?

File: openstack/compute/v2/extensions/hypervisors/doc.go [hunk 1]
```diff
@@ -24,7 +24,28 @@ Example of Show Hypervisor Details with Compute API microversion greater than 2.
...
+	allHypervisors, err := hypervisors.ExtractHypervisors(allPages)
+	if err != nil {
+		panic(err)
+	}
+
+	for _, hypervisor := range allHypervisors {
+		fmt.Printf("%+v\n", hypervisor)
+	}
+
+Example of Retrieving Details of All Hypervisors with Compute API microversion greater than 2.33
```
Comment (kayrus) [inline openstack/compute/v2/extensions/hypervisors/doc.go:41]: Not clear which microversion end user needs to use. 2.33 or 2.53 (from the acceptance tests)

**Review by kayrus — COMMENTED** [review]
I don't have a use case for this API call, I can only try to run acceptance tests.
The only thing that I have concerns on is an API version in docs.

File: openstack/compute/v2/extensions/hypervisors/doc.go (hunk 1 above)
Comment (jtopjian) [reply openstack/compute/v2/extensions/hypervisors/doc.go:41]: It is confusing, yes. The newly added `ListOpts` includes options that are valid in either 2.33 or 2.53. The title of this example is supposed to denote that microversion 2.33+ is required.

**Pull request merged**